functions/openhab.bash

```diff
@@ -143,12 +143,12 @@ openhab_shell_interfaces() {
 
   if [[ -n $INTERACTIVE ]]; then
     while [[ -z $sshPass ]]; do
-      if ! sshPass1="$(whiptail --title "Authentication setup" --passwordbox "$introText" 15 80 3>&1 1>&2 2>&3)"; then echo "CANCELED"; return 0; fi
+      if ! sshPass1="$(whiptail --title "Authentication setup" --passwordbox "$introText" 15 80 "$sshPass1" 3>&1 1>&2 2>&3)"; then echo "CANCELED"; return 0; fi
       if ! sshPass2="$(whiptail --title "Authentication setup" --passwordbox "\\nPlease confirm the password:" 9 80 3>&1 1>&2 2>&3)"; then echo "CANCELED"; return 0; fi
-      if [[ $sshPass1 == "$sshPass2" ]] && [[ ${#sshPass1} -ge 8 ]] && [[ ${#sshPass2} -ge 8 ]]; then
+      if [[ $sshPass1 == "$sshPass2" ]] && [[ ${#sshPass1} -ge 7 ]] && [[ ${#sshPass2} -ge 7 ]]; then
         sshPass="$sshPass1"
       else
-        whiptail --title "Authentication setup" --msgbox "Password mismatched, blank, or less than 8 characters... Please try again!" 7 80
+        whiptail --title "Authentication setup" --msgbox "Password mismatched, or less than 7 characters... Please try again!" 7 80
       fi
     done
   fi
```

Comment:
This will not work, it still requires interactive setup with the current code.

Reply:
if you want to define the password yes this requires interaction (just an instant idea: we could maybe reuse userpw)
but the more important part is the editing of .cfg and that should be done unattended during install

Reply:
What about this: no password set after unattended install but if run interactively it'll deploy the default password when user does not enter anything. To manually edit sshHost is a lot more hassle to users and for remote admins it's awkward without a console.